src/app/products/product-card-rating/product-card-rating.component.html

```diff
@@ -0,0 +1,9 @@
+<ul
+          class="list-inline rating-list"
+          *ngFor="let star of stars"
+          style="display: inline-block"
```

Comment:
move inline styling (style="") to .scss file 
<img width="670" alt="image" src="https://user-images.githubusercontent.com/26151759/177316799-e2b47f7c-e853-4cea-a392-5f7c73c14dcd.png">